Alignment/CommonAlignmentProducer/python/ALCARECOTkAlMuonIsolatedHI_cff.py

```diff
@@ -1,27 +1,21 @@
-# AlCaReco for track based alignment using isolated muon tracks
+# AlCaReco for track based alignment using isolated muon tracks - relaxed cuts for PbPb collisions
 import FWCore.ParameterSet.Config as cms
 
 from Alignment.CommonAlignmentProducer.ALCARECOTkAlMuonIsolated_cff import *
 
 ALCARECOTkAlMuonIsolatedHIHLT = ALCARECOTkAlMuonIsolatedHLT.clone(
-    eventSetupPathsKey = 'TkAlMuonIsolated'
+    eventSetupPathsKey = 'TkAlMuonIsolatedHI'
     )
 
 ALCARECOTkAlMuonIsolatedHIDCSFilter = ALCARECOTkAlMuonIsolatedDCSFilter.clone()
 
-ALCARECOTkAlMuonIsolatedHIGoodMuons = ALCARECOTkAlMuonIsolatedGoodMuons.clone()
-ALCARECOTkAlMuonIsolatedHIRelCombIsoMuons = ALCARECOTkAlMuonIsolatedRelCombIsoMuons.clone(
-    src = 'ALCARECOTkAlMuonIsolatedHIGoodMuons'
-)
-
 ALCARECOTkAlMuonIsolatedHI = ALCARECOTkAlMuonIsolated.clone(
```

Comment:
@jrcastle with this being remove you cannot let it in the path: https://github.com/cms-sw/cmssw/pull/16155/files#diff-9a1d4158ab5e79a6d2b1fe963ef4712cR29

resulting in:

```
raceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/2016-42/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_1_X_2016-10-09-0000/bin/slc6_amd64_gcc530/cmsDriver.py", line 55, in <module>
    run()
  File "/cvmfs/cms-ib.cern.ch/2016-42/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_1_X_2016-10-09-0000/bin/slc6_amd64_gcc530/cmsDriver.py", line 27, in run
    configBuilder.prepare()
  File "/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_1_X_2016-10-09-2300/python/Configuration/Applications/ConfigBuilder.py", line 2052, in prepare
    self.addStandardSequences()
  File "/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_1_X_2016-10-09-2300/python/Configuration/Applications/ConfigBuilder.py", line 745, in addStandardSequences
    getattr(self,"prepare_"+stepName)(sequence = '+'.join(stepSpec))
  File "/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_1_X_2016-10-09-2300/python/Configuration/Applications/ConfigBuilder.py", line 1240, in prepare_ALCA
    alcaConfig=self.loadDefaultOrSpecifiedCFF(sequence,self.ALCADefaultCFF)
  File "/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_1_X_2016-10-09-2300/python/Configuration/Applications/ConfigBuilder.py", line 1192, in loadDefaultOrSpecifiedCFF
    l=self.loadAndRemember(defaultCFF,unsch)
  File "/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_1_X_2016-10-09-2300/python/Configuration/Applications/ConfigBuilder.py", line 325, in loadAndRemember
    self.process.load(includeFile)
  File "/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_1_X_2016-10-09-2300/python/FWCore/ParameterSet/Config.py", line 519, in load
    module = __import__(moduleName)
  File "/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-10-09-2300/python/Configuration/StandardSequences/AlCaRecoStreams_cff.py", line 140, in <module>
    from DQMOffline.Configuration.AlCaRecoDQM_cff import *
  File "/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-10-09-2300/python/DQMOffline/Configuration/AlCaRecoDQM_cff.py", line 8, in <module>
    from DQMOffline.Alignment.ALCARECOTkAlDQM_cff import *
  File "/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-10-09-2300/python/DQMOffline/Alignment/ALCARECOTkAlDQM_cff.py", line 487, in <module>
    from Alignment.CommonAlignmentProducer.ALCARECOTkAlMuonIsolatedHI_cff import ALCARECOTkAlMuonIsolatedHIHLT
  File "/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-10-09-2300/python/Alignment/CommonAlignmentProducer/ALCARECOTkAlMuonIsolatedHI_cff.py", line 30, in <module>
    +ALCARECOTkAlMuonIsolatedHI
NameError: name 'ALCARECOTkAlMuonIsolatedHIGoodMuons' is not defined

```